File: 프로그래머스/unrated/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
 
        int max = 0;
        int maxIdx = -1;
        for(int i = 0 ; i<array.length; i++){
            if(max < array[i]){
                max = array[i];
                maxIdx = i;
            }
        }
        int[] answer = {max,maxIdx};
        return answer;
    }
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.js
function solution(array) {
    // 가장 큰 수 찾기
    const maxNum = Math.max(...array);
    
    // 그 수의 인덱스 찾기
    const index = array.indexOf(maxNum);
    
    // 결과를 배열로 반환
    return [maxNum, index];
}
